glue/__init__.py

```diff
@@ -118,6 +118,11 @@ def make_selector_func(roi):
     from .qt.custom_viewer import CustomViewer
     return CustomViewer.create_new_subclass(name, **kwargs)
 
+# Load built-in plugins
+from .config import load_plugins
+load_plugins()
```

Comment:
Last week I was playing around with the idea of using a lazily-loaded module. Instead of having to worry about when to import the plugins/config modules, we could set them up to transparently import the first time their modules are accessed. IOW, something like

```
import glue.config  # nothing happens here
...
glue.config.plugins  # plugins is a property which calls load_plugins and caches the result
```

That would simplify some of the logic of when something can/cant be imported, and delay slow imports until something is actually needed. 

You need to do a little hacking to get that to work -- http://stackoverflow.com/a/880550 . I wasn't convinced it was worth the magic, but I wonder what you think?